src/clj/test/cards-agendas.clj

```diff
@@ -227,6 +227,24 @@
         (core/score state :corp {:card (refresh napd)})
         (is (= 2 (:agenda-point (get-corp))) "Scored NAPD for 2 points after 5 advancements"))))
 
+(deftest nisei-mk-ii-step-43
```

Comment:
It's not his fault. I originally wrote the code as "phase 1.2" / "phase 4.3", but then discovered the real terminology from FFG is "Step 1.2". At that point I didn't feel comfortable changing all the code references.